TCMSApp/src/client/app/landing/landing.controller.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('app.landing')
+        .controller('LandingController', LandingController);
+
+    LandingController.$inject = ['logger', 'authservice', '$state', '$rootScope'];
+    /* @ngInject */
+    function LandingController(logger, authservice, $state, $rootScope) {
+        var vmLanding = this;
+        vmLanding.logIn = logIn;
+
+        activate();
+
+        function logIn(){
+            var Trello = authservice.authorize();
+            if(Trello.authorized()){
+                $state.go('dashboard');
+            }
+            $rootScope.$on('authorized', function(){
```

Comment:
why $rootScope.$on is here? as I understand it is an adding of event listener for 'authorized' event being broadcasted. do we need to add it only in the log in function? 

here  https://toddmotto.com/all-about-angulars-emit-broadcast-on-publish-subscribing/ is a nice article